reana/cli.py

```diff
@@ -1086,9 +1086,10 @@ def docker_build(user, tag, component, build_arg,
                          * (7) special value 'ALL' that will expand to include
                                all REANA repositories.
     :param user: DockerHub organisation or user name. [default=reanahub]
-    :param tag: Docker tag to use. Automatically filled with the output of
-        `git describe --dirty` if not provided.
-    :param build_arg: Optional docker build argument. (e.g. DEBUG=true)
+    :param tag: Docker tag to use. If ``dirty`` is passed as a value the
+        output of ``git describe --dirty`` will be used, i.e.
```

Comment:
Beware about tag option defaults, and note the discrepancy with the CLI help message above, we need to make them equal.

Also, note that if you happen to be exactly on a checked-out tag, e.g. `v0.5.0` just before releasing, then "describe --dirty" will produce a very "clean" version.  So it may look a bit weird to pass "dirty" as a value when releasing... We could rather name it "auto" (generated) or something, so that difference would be "latest vs auto" rather than "latest vs dirty".